src/styles/layout-grid.scss

```diff
@@ -120,20 +125,20 @@ $cols: 12;
 @media (min-width: 600px) {
   .#{$blockContainer} {
     padding: 0 $horizontal-gutter;
-  }
 
-  .#{$blockRow} {
-    @include fd-set-margins-x-equal(-$horizontal-gutter);
+    > .#{$blockRow} {
+      @include fd-set-margins-x-equal(-$horizontal-gutter);
 
-    min-width: calc(100% + #{$horizontal-gutter} * 2);
+      min-width: calc(100% + #{$horizontal-gutter} * 2);
 
-    .#{$blockCol} {
-      @include fd-set-paddings-y-equal(math.div($vertical-gutter, 2));
+      > .#{$blockCol} {
+        @include fd-set-paddings-y-equal(math.div($vertical-gutter, 2));
+      }
```

Comment:
this whole block of code is very similar to the one from lines 87-94 and can be moved into a mixin with parameters for x and y spacing